src/renderer/api/kube-object.ts

```diff
@@ -122,12 +122,15 @@ export class KubeObject implements ItemObject {
     return this.metadata.namespace || undefined;
   }
 
-  // todo: refactor with named arguments
-  getAge(humanize = true, compact = true, fromNow = false) {
+  getTimeDiffFromNow(): number {
+    return new Date().getTime() - new Date(this.metadata.creationTimestamp).getTime();
```

Comment:
No need to create a new `Date` instance if all that is being done is calling `getTime()`

```suggestion
    return Date.now() - new Date(this.metadata.creationTimestamp).getTime();
```

Reply:
I guess technically it's the same :) So, not a big deal.

Reply:
True but it is another allocation (I think)

Reply:
`Date.now()` is clearer and more concise

Reply:
Okay, you are right, guys. 
Will make a separate PR for that (to avoid reset approvals / speed up code-flow)